tests/sqllogictests/suites/base/20+_others/20_0013_query_result_cache.test

```diff
@@ -13,6 +13,9 @@ CREATE TABLE IF NOT EXISTS t1 (a INT not null);
 statement ok
 INSERT INTO t1 VALUES (1), (2), (3);
 
+statement ok
```

Comment:
The size of the result cache after parallel sorting is variable, sometimes 117 and sometimes 125.